src/tests/system/tests/test_feature_presence.py

```diff
@@ -0,0 +1,86 @@
+"""
+SSSD Feature presence suite
+
+:requirement: features
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.parametrize(
+    "distribution, distro_major, distro_minor, sssd_major, sssd_minor, feature, presence",
+    [
+        ("Fedora", 39, 0, 2, 9, "passkey", True),
+        ("CentOS Stream", 9, 0, 2, 9, "passkey", True),
+        ("Red Hat Enterprise Linux", 9, 4, 2, 9, "passkey", True),
+        ("Ubuntu", 23, 10, 2, 9, "passkey", True),
+        (None, None, None, 2, 10, "knownhosts", True),
+        ("Fedora", 40, 0, 2, 10, "files-provider", False),
+        ("CentOS Stream", 10, 0, 2, 10, "files-provider", False),
+        ("Red Hat Enterprise Linux", 10, 0, 2, 10, "files-provider", False),
+        (None, None, None, 2, 10, "ldap_use_ppolicy", True),
+        ("Fedora", 41, 0, 2, 10, "non-privileged", True),
+        ("CentOS Stream", 9, 0, 2, 10, "non-privileged", True),
+        ("Red Hat Enterprise Linux", 9, 4, 2, 10, "non-privileged", True),
+    ],
+)
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopology.Client)
+def test_feature__feature_presence(
```

Comment:
The file should be renamed to 'test_feature.py' then the test name can be 'test_feature__is_present()' which is more readable.

Reply:
I renamed the file to `test_feature.py`, but I don't agree with the test name. The test not only checks that a feature is present, but also that it is not present. For example, it checks that `files-provider` is not present after a certain version. This is why I think the current name is more correct.

Reply:
how about 'test_feature__presence' ?

Reply:
Sounds good to me so I updated it